src/hooks/useAccount.ts

```diff
@@ -152,8 +152,7 @@ export function useAccount<T>(
       disposeEmitter()
       setTimeout(disposeWatch, 30 * 1000) // Keep cached accounts around for 30s in case a rerender is causing reuse
     }
-  }, []) // only trigger on change to parser if it wasn't defined before.
-
+  }, [cache, id, !parser]) // only trigger on change to parser if it wasn't defined before.
```

Comment:
Did we want to change this? Comment indicates no?

Reply:
This will eventually go away once my PR goes into `helium/programming-library` that fixes some of the hooks for `react-native`